src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsActionGroupEmailReceiver.ps1

```diff
@@ -0,0 +1,29 @@
+function Add-ArmApplicationInsightsActionGroupEmailReceiver {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationInsightsActionGroup")]
+    Param(
+        [PSTypeName("ApplicationInsightsActionGroup")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ActionGroup,
+        [Parameter(Mandatory)]
+        [string]
+        $Name,
+        [Parameter(Mandatory)]
+        [ValidatePattern("^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-0-9a-z]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$")]
```

Comment:
email regex always has exception, i wouldnt try to validate it.

https://stackoverflow.com/questions/201323/how-to-validate-an-email-address-using-a-regular-expression